.github/workflows/pull_request_release.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - run: git fetch --prune --unshallow
```

Comment:
This provides the action with access to all the tags and git history, allowing changelog to be generated with all changes going into the release